### DISCUSSION TITLE
Deactivate command autocompletion

### DISCUSSION THREAD

**Discussion by olischma** [Minestom/Minestom#1367] [Help] [answered]
Hey, 

I'm currently trying to disable autocompletion for a specific command in Minestom. My goal is to create an easter egg command, which should not be visible by tabbing trough commands...

Any ideas on this?

Thanks!

**Comment by GoldenStack** [comment, marked as answer]
You can set the command's condition (Command#setCondition) to something that returns false if the provided string is null, and true otherwise.
This works because the `commandString` argument in the conditions is only null if it is being queried to see if it should be sent to the player, and it will have the actual command they are executing otherwise.
See here for more information:
https://javadoc.minestom.net/net/minestom/server/command/builder/condition/CommandCondition.html

**Reply by olischma** [reply]
Thanks, that worked fine!

`setCondition((sender, commandString) -> commandString != null);`

now the command does not show up in the command list but still can be executed without problems!
